build_bolt.bash

```diff
@@ -3,21 +3,13 @@
 mkdir -p build-bolt || (echo "Could not create build-bolt directory"; exit 1)
 cd build-bolt
 
-echo "== Configure build for only llvm-bolt (facebookincubator project)"
-echo "== Build with host-compiler, not part of the bootstrapping"
 echo "== Deactivate everything but 'llvm-bolt'"
-
-cmake -G Ninja \
-	-DBUILD_SHARED_LIBS=OFF \
-	-DCMAKE_BUILD_TYPE=Release \
-	-DCMAKE_INSTALL_PREFIX="$(pwd)/install" \
+CC=clang CXX=clang++ LD=lld \
+	cmake -G Ninja ../llvm-project/llvm \
+	-DLLVM_ENABLE_PROJECTS="bolt" \
```

Comment:
bolt can be just put into `stage1`.
no need to do another compilation run since its upstreamed